Frontend/ui-customer/src/Strings/BN.js

```diff
@@ -427,7 +427,7 @@ const bengaliStrings = {
   "NO_DONT" : "না, না",
   "YES_CANCEL_SEARCH" : "হ্যাঁ, অনুসন্ধান বাতিল করুন",
   "TRY_LOOKING_FOR_RIDES_AGAIN" : "এটি খুব ব্যস্ত দিন বলে মনে হচ্ছে। আপনি আবার রাইড খুঁজছেন চেষ্টা করতে পারেন",
-  "NO_TIP" : "0 টিপ্",
+  "NO_TIP" : "শূন্য টিপ",
```

Comment:
@AmritpalSingh6749 can we revert this change and add rupee symbol in front of 0 as this change was discussed with  sreedeep.